config/routes.rb

```diff
@@ -1,3 +1,9 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
- [ ] Let's remove all the non-resourceful routes since we don't need them. We only need the `resources :users`.
- [ ] In this appliacation we are not going to use all the actions for users routes. Let's think of what action do we need for this route. Check out [this](https://guides.rubyonrails.org/routing.html#resources-on-the-web) please.